server/db/models/class.ts

```diff
@@ -0,0 +1,55 @@
+import mongoose from 'mongoose'
+
+const { Schema } = mongoose
+
+interface IClass {
+    title: string //Title of the class
+    description: string //Description of the class
+    order: number //If is the first class of the course, or the second...
+    file: string //ID of the file in the blob-storage database
+}
+
+interface ClassDoc extends mongoose.Document {
+    title: string
+    description: string
+    order: number
+    file: string
+}
+
+interface ClassModelInterface extends mongoose.Model<ClassDoc> {
+    build(attr: IClass): ClassDoc
+}
+
+const classSchema = new Schema({
+    title: {
+        type: String,
+        required: true,
+        minLength: 1,
+        maxLength: 140,
+    },
+    description: {
+        type: String,
+        required: true,
+        minLength: 1,
+        maxLength: 520,
+    },
+    order: {
+        type: Number,
+        required: true,
+        min: 1,
+    },
+    file: {
+        type: String,
+        required: true,
+    },
+});
+
```

Comment:
I think here you should transform json representation of class, just like we did in material model. The rest of the code smells so good!